functions/functions_fundamentals.py

```diff
@@ -0,0 +1,35 @@
+def greetings(): #defining a function
+    name = input("Wie heißen Sie? ")
+    print(f"Hallo {name}, Guten tag!")
+
+greetings() # calling the function
+
+
+def add_numbers(a,b): # defining a function with parameters
+    print(f"sum is {a+b}")
+
+add_numbers(4,6) #6 and 4 are passed as arguments to the function add_numbers
+
+
+def divide_numbers(x,y): #with return statement
+    if y == 0:
+        return f"Division by zero is not allowed."
```

Comment:
you should not return if it's 0 rather raise an error using builtin error
```suggestion
        raise ZeroDivisionError("Cannot divide by zero!")
```